src/controllers/album.js

```diff
@@ -0,0 +1,97 @@
+const db = require('../../src/db/index.js');
+
+const createAlbum = async (req, res) => {
+  const { name, year } = req.body;
+  const { id } = req.params;
+
+  try {
+    const {
+      rows: [album],
+    } = await db.query(
+      'INSERT INTO Albums (name, year, artistid) VALUES ($1, $2, $3) RETURNING *',
+      [name, year, id]
+    );
+    res.status(201).json(album);
+  } catch (err) {
+    console.log(err);
```

Comment:
Since youre returning an error here already is the console log actually needed? Anywhere youre already returning something as you are here you can get rid of the console log as its surplus to requirements